manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -1592,7 +1592,6 @@ def _enable_nfs_protocols(self, versions):
             nfs_service_modify_args['is-nfsv40-enabled'] = 'true'
         if 'nfs4.1' in versions:
             nfs41_opts = {
-                'is-nfsv41-pnfs-enabled': 'true',
                 'is-nfsv41-acl-enabled': 'true',
                 'is-nfsv41-read-delegation-enabled': 'true',
                 'is-nfsv41-write-delegation-enabled': 'true',
```

Comment:
I tracked the refs to this  function `_enable_nfs_protocols()`, it goes all the way back to `ensure_share_server`. So the call stack is `ensure_share_server` -> `update_server` -> `_update_vserver` -> 'enable_nfs` ->  `enable_nfs_protocols`. It means that the existing vservers will be modified in the manila ensure process? Can that be harmful for mounted shares?